server/features/content_copy.feature

```diff
@@ -28,7 +28,7 @@ Feature: Copy Content in Personal Workspace
       {"state": "draft"}
       """
       When we get "/archive/#copy._id#?version=all"
-      Then we get list with 3 items
+      Then we get list with 4 items
       When we get "/archive/"
```

Comment:
As per the Scenario, there should be 3 items only in archive_versions. Because of the changes in `duplicate_item()` in `archive.py` module we see 4 items which is not desired behavior.